.drone.yml

```diff
@@ -59,6 +59,7 @@ pipeline:
     image: docker/compose:1.17.1
     commands:
       - cd docker
+      - cp .env.dist .env
```

Comment:
seems we should pass params directly. Like we already to at SE.

I removed `.env.dist` at https://github.com/fidals/stroyprombeton/pull/195